manifests/server/installdb.pp

```diff
@@ -19,7 +19,7 @@
       creates   => "${datadir}/mysql",
       logoutput => on_failure,
       path      => '/bin:/sbin:/usr/bin:/usr/sbin:/usr/local/bin:/usr/local/sbin',
-      require   => Package[$mysql::server::package_name],
+      require   => Package['mysql-server'],
```

Comment:
But if `$mysql::server::package_manage` is false or undef and the package resource isn't created, then this should rightfully fail right? But looking at the code, it is set from `$mysql::params::server_package_name` which should always be set.